pages/common/wondershaper.md

```diff
@@ -0,0 +1,28 @@
+# wondershaper
+
+> Wonder Shaper is a script that allows the user to limit the bandwidth of one or more network adapters.
+> More information: <https://github.com/magnific0/wondershaper#usage>.
+
+- Display help:
+
+`wondershaper -h`
```

Comment:
```suggestion
- Print the [h]elp:

`wondershaper -h`
```

Reply:
It's better to highlight the first character of the short option in the command description.